keylime/cmd/attest.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright 2017 Massachusetts Institute of Technology.
+"""
+
+from keylime import keylime_logging
+from keylime.da import attest
+
+logger = keylime_logging.init_logging("attest")
+
+
+def main():
```

Comment:
If you annotated these new files right away it may be beneficial

Reply:
done